### DISCUSSION TITLE
the userpass under auth not work?

### DISCUSSION THREAD

**Discussion by forsetiwang** [apernet/hysteria#699] [General]
version: 2.0.2 
server side config:
auth:
  type: password
  password: test001
  userpass:
    testuser1: 111111
 
 client side config:
 auth:  testuser1:111111
   
this not working, is this version 2.0.2 support this or need other config?

**Comment by tobyxdd** [comment]
`type: password` should be `type: userpass`

**Reply by AkaneNakamoto** [reply]
the documentation should be corrected then: https://v2.hysteria.network/zh/docs/advanced/Full-Server-Config/

**Reply by tobyxdd** [reply]
> the documentation should be corrected then: https://v2.hysteria.network/zh/docs/advanced/Full-Server-Config/

The document includes all 4 possible methods. You are supposed to change `type` to the one you want to use.

**Reply by AkaneNakamoto** [reply]
from the user's perspective it's kind of hard to figure out the need to change 'type' by simply reading the current doc, a pr was just submitted for it https://github.com/apernet/hysteria-website/pull/32

**Comment by forsetiwang** [comment]
You are great. It works now. Thanks!
